src/index.html

```diff
@@ -7,12 +7,65 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <!-- Font -->
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+    <!--  -->
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <section
+      class="card"
+      data-qa="card"
+    >
+      <img
+        src="images/imac.jpeg"
+        alt="Desctop-img"
+        class="card__img card__img--margin"
+      />
+
+      <h1 class="card__title card__title--margin">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h1>
+      <p class="card__description card__description--margin">
+        Product code: 195434
+      </p>
+
+      <div class="card__rating card__rating--margin">
+        <div class="stars">
```

Comment:
Add the modifier class
```suggestion
        <div class="stars stars--4">
```